### PR TITLE
Add contribution instructions, code of conduct, and codeowners

### PR DESCRIPTION
**Pull request by bensheldon**
Connects to #109.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+We welcome your contributions to the project. Thank you!
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+
+## What to contribute
+
+This repository, `rubocop-github` is part of a broader RuboCop ecosystem.
+
+If the Cop you would like to propose is **generally applicable outside of GitHub**:
```
Comment (issyl0) [inline CONTRIBUTING.md:12]: Should we be stronger here about only allowing contributions from GitHub staff? Maybe something like "we only accept new cop contributions if there's evidence they're applicable to and useful for GitHub's codebase".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+We welcome your contributions to the project. Thank you!
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+
+## What to contribute
+
+This repository, `rubocop-github` is part of a broader RuboCop ecosystem.
```
Comment (sampart) [inline CONTRIBUTING.md:10]:
```suggestion
This repository, `rubocop-github`, is part of a broader RuboCop ecosystem.
```

**Review by sampart — COMMENTED** [review]
Thanks so much for moving this forward ❤️ Just a quick grammar suggestion.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bensheldon) [reply CONTRIBUTING.md:12]: good flag! I'm willing to defer that until we see it. I'm not too worried that we'll be overwhelmed by contributions 😆

**Pull request merged**